api-admin/src/main/java/bio/terra/pearl/api/admin/controller/GlobalExceptionHandler.java

```diff
@@ -50,13 +48,20 @@ public ResponseEntity<ErrorReport> authenticationExceptionHandler(Exception ex)
     return buildErrorReport(ex, HttpStatus.UNAUTHORIZED, request);
   }
 
-  @ExceptionHandler({
-    UnauthorizedException.class,
-  })
+  @ExceptionHandler({UnauthorizedException.class, PermissionDeniedException.class})
   public ResponseEntity<ErrorReport> authorizationExceptionHandler(Exception ex) {
     return buildErrorReport(ex, HttpStatus.FORBIDDEN, request);
   }
 
+  @ExceptionHandler({
+    NotFoundException.class,
+    javax.ws.rs.NotFoundException.class,
+    bio.terra.pearl.core.service.exception.NotFoundException.class
```

Comment:
That's a lot of NotFound variations!
I guess it's too late at this point but is there a reason Juniper invented it's own (and based other exception classes on that)?

Reply:
that might be worth an infrastructure ticket to investigate whether we should just say "we all know core is probably on ever getting used only in a webapp, so it might as well import some web packages to avoid having an extra translation layer"

Reply:
https://broadworkbench.atlassian.net/browse/JN-737